### PR TITLE
Trust Stores files updates

### PR DESCRIPTION
**Pull request by clstmbrly**
@lachellel Some minor edits to newly updated Trust Stores files--editorial, consistency, display.  One question I had was whether to use "Windows operating systems" vs. "Microsoft operating systems."  Where we say "Windows, Apple, and Android operating systems," it seems more correct to say "Microsoft" (albeit Android is an OS on many device types).  Please let me know your preferences or if any feedback.  Thanks!
@ryancdickson

### CODE REVIEW

File: _truststores/01_msft_removal_solutions.md [hunk 1]
```diff
@@ -5,17 +5,15 @@ title: Microsoft Removal Solutions
 collection: truststores
 permalink: truststores/microsoft/
 ---
-In FY 2019, the U.S. Government is removing the Federal Public Key Infrastructure (FPKI) root certificate from the Microsoft certificate store. This change will affect all federal agencies and may have an impact on the following services:  
+In FY 2019, the U.S. Government is removing the Federal Public Key Infrastructure (FPKI) root certificate from the Microsoft certificate Trust Store. This change will affect all federal agencies and may have an impact on the following services:
```
Comment (lachellel) [inline _truststores/01_msft_removal_solutions.md]: revert this change please.  we agreed upon certificate store for this announcement based on the common terms used in general microsoft technet searches.

File: _truststores/01_msft_removal_solutions.md [hunk 2]
```diff
@@ -5,17 +5,15 @@ title: Microsoft Removal Solutions
...
 ---
-In FY 2019, the U.S. Government is removing the Federal Public Key Infrastructure (FPKI) root certificate from the Microsoft certificate store. This change will affect all federal agencies and may have an impact on the following services:  
+In FY 2019, the U.S. Government is removing the Federal Public Key Infrastructure (FPKI) root certificate from the Microsoft certificate Trust Store. This change will affect all federal agencies and may have an impact on the following services:  
  
 - Personal Identity Verification (PIV) credential authentication to the government networks
 - Agency web applications implementing client authentication (e.g., PIV authentication)
 - Authentication to Office 365
 - Validation of digital signatures
-- Other applications leveraging the Microsoft certificate store
+- Other applications leveraging the Microsoft certificate Trust Store
```
Comment (lachellel) [inline _truststores/01_msft_removal_solutions.md]: revert please

File: _truststores/00_index.md [hunk 3]
```diff
@@ -5,78 +5,74 @@ collection: truststores
...
 ---
 
-This is a high-level overview of what application trust stores are and a list of commonly used application trust stores and settings.  
+This is a high-level overview of what application trust stores are and a list of commonly used trust stores and settings.  
 
 * [What is a Trust Store?](#what-is-a-trust-store)
 * [What are the most commonly used Trust Stores?](#what-are-the-most-commonly-used-trust-stores)
-* [How do I set dynamic path validation for the trust store in Windows operating systems?](#how-do-i-set-dynamic-path-validation-for-the-trust-store-in-windows-operating-systems)
-* [How do I check which Federal PKI certificate policies are trusted by Adobe?](#how-do-i-check-which-federal-pki-certificate-policies-are-trusted-by-adobe)
+* [How do I set dynamic path validation for the Trust Store in Windows operating systems?](#how-do-i-set-dynamic-path-validation-for-the-trust-store-in-windows-operating-systems)
```
Comment (lachellel) [inline _truststores/00_index.md]: make trust store lower case everywhere vs upper case

**Review by lachellel — CHANGES_REQUESTED** [review]

File: _truststores/01_msft_removal_solutions.md (hunk 1 above)
Comment (clstmbrly) [reply _truststores/01_msft_removal_solutions.md]: Will do.  Thanks

File: _truststores/00_index.md (hunk 3 above)
Comment (clstmbrly) [reply _truststores/00_index.md]: Will fix all instances.  Thanks

Comment (clstmbrly) [reply _truststores/00_index.md]: For the main announcement (00_index.md), "trust store" references should **not** be changed to "certificate store"?  Many trust store refs in it.

Comment (lachellel) [reply _truststores/00_index.md]: nope; for the main page, the trust store references should not be changed to certificate store.

Comment (clstmbrly) [reply _truststores/00_index.md]: Got it.  Thanks.

**Review by lachellel — APPROVED** [review]
